src/main/java/de/cronn/diff/html/HtmlBuilder.java

```diff
@@ -22,33 +22,33 @@
 
 	private static final String CSS_FILE = "diffToHtml.css";
 
-	protected static final String EMPTY_LINE_HEIGHT = "16";
+	static final String EMPTY_LINE_HEIGHT = "16";
 
-	protected static final String CSS_CLASS_LINE_NUMBER = "line-number";
+	static final String CSS_CLASS_LINE_NUMBER = "line-number";
```

Comment:
Why? why? and below more why?s

Reply:
Actually, if you want to change it, these constants could/should all be `public`.